content/en/02-installing-pixie/04-install-schemes/01-cli.md

```diff
@@ -26,11 +26,18 @@ bash -c "$(curl -fsSL https://withpixie.ai/install.sh)"
 
 ### Directly downloading the binary
 
+- [https://storage.googleapis.com/pixie-dev-public/cli/latest/cli_darwin_universal](https://storage.googleapis.com/pixie-dev-public/cli/latest/cli_darwin_universal)
+- [https://storage.googleapis.com/pixie-dev-public/cli/latest/cli_darwin_universal.sha256](https://storage.googleapis.com/pixie-dev-public/cli/latest/cli_darwin_universal.sha256)
+- [https://storage.googleapis.com/pixie-dev-public/cli/latest/cli_linux_amd64](https://storage.googleapis.com/pixie-dev-public/cli/latest/cli_linux_amd64)
+- [https://storage.googleapis.com/pixie-dev-public/cli/latest/cli_linux_amd64.sha256](https://storage.googleapis.com/pixie-dev-public/cli/latest/cli_linux_amd64.sha256)
+
```

Comment:
this looks a bit out of place, maybe perface it and/or shorten the links?
Github names this section `assets` and usually only includes the last part of the part as the link text.
For example: https://github.com/derailed/k9s/releases/tag/v0.25.18